### PR TITLE
{Release} Upgrade to Azure CLI 2.25.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,102 @@
...
+2.25.0
+++++++
+
+**ACR**
+
+* `az acr connected-registry`: Minor bug fixes (#18288)
+
+**App Service**
+
+* :  Fixing az webapp deployment source config-local-git to set SiteConfig (#18364)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:15]:
```suggestion
* Fix az webapp deployment source config-local-git to set SiteConfig (#18364)
```

File: src/azure-cli-core/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.25.0
+++++++
+* Fix #18360 and #18262: write to stderr issue (#18414)
```
Comment (zhoxing-ms) [inline src/azure-cli-core/HISTORY.rst:8]:
```suggestion
* Fix #18360 and #18262: Write to stderr issue (#18414)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,102 @@
...
+
+**Compute**
+
+* Upgrade api-version for VM and VMSS from '2020-12-01' to '2021-03-01' (#18233)
+* `az vm create`: Support delete option for NICs and Disks for VMs in Azure CLI (#18238)
+* Support user_data for VM and VM Scale Sets (#18432)
+
+**Container**
+
+* `az container exec`: decode received bytes as utf-8 string (#18384)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:35]:
```suggestion
* `az container exec`: Decode received bytes as utf-8 string (#18384)
```

Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:35]:
```suggestion
* `az container exec`: Decode received bytes as utf-8 string (#18384)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,102 @@
...
+
+**EventGrid**
+
+* Migrate track2 SDK (#18210)
+
+**HDInsight**
+
+* Migrate to track2 python sdk 7.0.0 (#18237)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:43]:
```suggestion
* Migrate to track2 Python SDK 7.0.0 (#18237)
```

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:15]:
```suggestion
* `az webapp deployment source config-local-git`:  Fixing to set SiteConfig (#18364)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,102 @@
...
+**SignalR**
+
+* `az signalr identity`: Add managed identity related command (#18309)
+* `az signalr cors update`: Add update command for cors (#18309)
+
+**Storage**
+
+* `az storage blob copy start`: Support --tier and --rehydrate-priority (#18170)
+* GA release storage file share NFS and SMB multichannel (#18232)
+* BREAKING CAHNGE: `az storage account create`: Remove `StorageFileDataSmbShareOwner` option for --default-share-permission (#18396)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:90]:
```suggestion
* [BREAKING CAHNGE] `az storage account create`: Remove `StorageFileDataSmbShareOwner` option for --default-share-permission (#18396)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,102 @@
...
+**RDBMS**
+
+* Add commands for Github actions (#17949)
+* `az postgres flexible-server migration`: Add customer facing feature to migrate postgres db servers from Sterling to Meru platform (#18161)
+* Private DNS zone parameter added for restore command, high availability validator (#18218)
+* Change server default location (issue reported) (#18157)
+
+**Role**
+
+* [BREAKING CHANGE]  `az ad sp create-for-rbac`: `--name` is now only used as the `displayName` of the app. It is not used to generate `identifierUris` anymore. `name` in the output is now the same as `appID` (`servicePrincipalNames`) and deprecated. (#18312)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:79]:
```suggestion
* [BREAKING CHANGE] `az ad sp create-for-rbac`: `--name` is now only used as the `displayName` of the app. It is not used to generate `identifierUris` anymore. `name` in the output is now the same as `appID` (`servicePrincipalNames`) and deprecated. (#18312)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,102 @@
...
+**NetAppFiles**
+
+* `az netappfiles volume backup status`: Command added to get the status of the backup for a volume. (#18303)
+* `az netappfiles volume update`: Optional parameter added named `--snapshot-policy-id` o assign a snapshot policy to the volume. (#18303)
+* `az netappfiles volume backup create`: Optional parameter added named `--use-existing-snapshot` to manually backup an already existing snapshot. (#18303)
+* `az netappfiles volume backup update`: Optional parameters added named `--use-existing-snapshot` to manually backup an already existing snapshot. Optional parameter label also added to add a label to backup. (#18303)
+
+**Network**
+
+* Private link add `Microsoft.Sql/servers` provider (#18268)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:66]:
```suggestion
* Private link supports `Microsoft.Sql/servers` provider (#18268)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,102 @@
...
+**Storage**
+
+* `az storage blob copy start`: Support --tier and --rehydrate-priority (#18170)
+* GA release storage file share NFS and SMB multichannel (#18232)
+* BREAKING CAHNGE: `az storage account create`: Remove `StorageFileDataSmbShareOwner` option for --default-share-permission (#18396)
+* `az storage blob list`: --delimiter parameter value will now be honored (#18394)
+
+**Synapse**
+
+* Update to  AZ Synapse mgmt 2.0.0 (#18195)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:95]:
```suggestion
* Update to AZ Synapse mgmt 2.0.0 (#18195)
```

Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:95]:
```suggestion
* Update `azure-mgmt-synapse` to 2.0.0 (#18195)
```

**Review by evelyn-ys — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Comment by yonzhan** [comment]
Release

**Pull request merged**